src/wp-includes/formatting.php

```diff
@@ -2943,6 +2943,10 @@ function _make_url_clickable_cb( $matches ) {
 		$suffix = $matches[3];
 	}
 
+	if ( isset( $matches[4] ) && ! empty( $matches[4] ) ) {
```

Comment:
```suggestion
	if ( ! empty( $matches[4] ) ) {
```

`empty()` also checks if the element is exists.